### PR TITLE
Add Wireless Transfer App.app Latest

### PR DESCRIPTION
**Pull request by nordringrayhide**

### CODE REVIEW

File: Casks/wirelesstransferapp.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+cask :v1 => 'wirelesstransferapp' do
+  version :latest
+  sha256 :no_check
+
+  url 'http://www.wirelesstransferapp.com/wirelesstransferapp-intel.dmg'
+  name 'Wireless Transfer App'
+  homepage 'http://www.wirelesstransferapp.com'
+  license :freemium
```
Comment (vitorgalvao) [inline Casks/wirelesstransferapp.rb]: Is this a free app that works indefinitely but with limitations that can be removed by paying?

Comment (romanvbabenko) [inline Casks/wirelesstransferapp.rb]: It's absolutely free

Comment (vitorgalvao) [inline Casks/wirelesstransferapp.rb]: [Freemium does not mean free](http://en.m.wikipedia.org/wiki/Freemium). What you want here is `:gratis`.

Comment (romanvbabenko) [inline Casks/wirelesstransferapp.rb]: Ooook. I've fixed license. What's next?

File: Casks/wirelesstransferapp.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
+cask :v1 => 'wirelesstransferapp' do
```
Comment (vitorgalvao) [inline Casks/wirelesstransferapp.rb]: Just noticed this needs to be renamed as `wireless-transfer`, to comply with the [token rules](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_token_reference.md)

Comment (romanvbabenko) [inline Casks/wirelesstransferapp.rb]: Done.

**Pull request merged**